### ISSUE TITLE
< span > fails color-contrast rule when inside a disabled button.

### ISSUE THREAD

**Issue by Vanals** [dequelabs/axe-core#2626]
Hi everyone,

In the project I am working I realized we were disabling the rule color-contrast because was being fired by disabled-buttons.

![Screenshot 2020-11-06 at 16 38 16](https://user-images.githubusercontent.com/25549355/98391370-7953b900-204e-11eb-92c6-17720fdebaac.png)

The HTML structure of the failing button is like so:
![screenshot_2020-11-06_at_15 32 20](https://user-images.githubusercontent.com/25549355/98391306-60e39e80-204e-11eb-91aa-4c066d189d44.png)

The element highlighted by cypress is the span:

![screenshot_2020-11-06_at_15 57 51](https://user-images.githubusercontent.com/25549355/98391341-6ccf6080-204e-11eb-88d4-49f5ec5154e7.png)

Expectation: The color-contrast rule should not fail as the span is inside a disabled button.

Motivation: A button is not supposed to respect color ration 4.5.: if disabled

    "axe-core": "^3.5.4",
    "cypress": "4.7.0",
    "cypress-axe": "^0.5.1",

upgrade the version to the latest released will require time, so if you can already spot the issue would be great. Thanks.

**Comment by straker** [comment]
Interesting. I'm not able to reproduce the issue with the following code:

```html
<button type="button" disabled style="background: #eee">
  <div>
    <div>
      <span style="color: #aaa">Button</span>
    </div>
  </div>
</button>
```

Would you be able to provide a working example that shows the violation happening?

**Comment by Vanals** [comment]
Thank you @straker  I will look into it and give you more information as soon as possible. A bit overwhelmed with other bits at the moment but  I will indeed get back to you! 

Thanks

**Comment by straker** [comment]
@Vanals were you able to get a chance to look further into this?

**Comment by WilcoFiers** [comment]
Since this issue has not had a response and we're unable to reproduce the problem, I'm closing the issue. If anyone can provide an example of the problem, feel free to open a new issue.
